### PR TITLE
Added functions for convolution and gaussian kernels in FloatVectorList.

### PR DESCRIPTION
**Pull request by niklasborwell**

### CODE REVIEW

**Comment by niklasborwell** [comment]
Should I worry about the failed travis build?

**Comment by davidhesselbom** [comment]
I don't think so.

> No output has been received in the last 10 minutes, this potentially indicates a stalled build or something wrong with the build itself.

I'll look into it when I have time.

File: source/math/FloatVectorList.ooc [hunk 1]
```diff
@@ -124,4 +125,52 @@ FloatVectorList: class extends VectorList<Float> {
...
+		result := This new(size)
+		for (i in 0..size)
+			result add(1.0f / (sqrt(2.0f * Float pi) * sigma) * pow(Float e, - 0.5f * ((i - (size - 1.0f) / 2.0f) squared()) / (sigma squared())))
+		sum := result sum
+		for (i in 0..size)
+			result[i] = result[i] / sum
+		result
+	}
+	forwardGaussianKernel: static func ~defaultSigma (size: Int) -> This {
+		This forwardGaussianKernel(size, size / 3.0f)
```
Comment (davidhesselbom) [inline source/math/FloatVectorList.ooc]: You need to cast `size` to `Float`, or `size / 3.0f` will get you an `Int`.

File: source/math/FloatVectorList.ooc [hunk 2]
```diff
@@ -124,4 +125,52 @@ FloatVectorList: class extends VectorList<Float> {
...
+		result := This gaussianKernel(size, sigma)
+		for (i in 0..(size - 1) / 2)
+			result[i] = 0.0f
+		sum := result sum
+		for (i in 0..size)
+			result[i] = result[i] / sum
+		result
+	}
+	backwardGaussianKernel: static func ~defaultSigma (size: Int) -> This {
+		This backwardGaussianKernel(size, size / 3.0f)
```
Comment (davidhesselbom) [inline source/math/FloatVectorList.ooc]: Same here...

**Pull request merged**